io.github.sxyazi.yazi.yml

```diff
@@ -0,0 +1,170 @@
+id: io.github.sxyazi.yazi
+runtime: org.freedesktop.Platform
+runtime-version: &runtime-version '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+  - org.freedesktop.Sdk.Extension.golang
+command: yazi
+add-extensions:
+  # Needs ffmpeg to generate video preview thumbnails
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: false
+    autodelete: true
+
+finish-args:
+  # For downloading plugins
+  - --share=network
+  # Use host-spawn to call utilities not packaged in the sandbox
+  - --talk-name=org.freedesktop.Flatpak
+  # File manager needs access to every file
+  - --filesystem=host
+  - --filesystem=home
+  - --filesystem=xdg-config/yazi:create
+  - --filesystem=xdg-data/yazi:create
+  - --filesystem=xdg-data/zoxide:create
```

Comment:
I'll remove the two entries for Yazi. The `zoxide` directory would be helpful for some users since they might be using zoxide in their shell alongside Yazi. It would be helpful for the zoxide database to be synced across Yazi and the shell.

Reply:
> zoxide directory would be helpful for some users since they might be using zoxide in their shell alongside Yazi.

then they can store the zoxide config in `~/.var/app/$app-id/config`. You cannot trust configs from host, the version of zoxide on host may not match with the one shipped in the flatpak.

It's entirely possible a newer version supports a config option that makes older versions error with "unrecognized config" or similar.

Reply:
Good point. I removed these permissions and added a note in the README.